src/babel-plugin-dynamic-import-amd.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
This made me wonder... what do we now advise users do for modules in their *build* pipeline? Why would a user use this AMD transform approach in their dev server, but something totally different in their build pipeline? (Or conversely, whatever it is they're doing in their build pipeline, why wouldn't they want the same thing in their dev server?)

Reply:
Well, that's more of an app builder consideration.  The initial use case for this middleware was lit-element and custom element library development, which don't compile modules out.

Right now we haven't solved for community alternative to the existing polymer build pipeline beyond "try webpack or rollup".

That's the next one to solve I think.

Reply:
Yep! It would be fantastic to eventually have a blog post that explains all the pieces that are going to survive from our tools, and what exactly we'll recommend people do for build, test, serve, etc. -- all the open-wc equivalents, etc.